trade-finance-manager-ui/templates/utilisation-reports/add-payment.njk

```diff
@@ -9,9 +9,14 @@
 {% import "./_macros/payment-date-input.njk" as paymentDateInput %}
 {% extends "index.njk" %}
 {% set backLink = { "href": backLinkHref } %}
-{% block pageTitle %}Add a payment{% endblock %}
+{% set pageHasErrors = errors.errorSummary | length %}
```

Comment:
Would this work?
```suggestion
{% set validationErrors = errors.errorSummary %}
```

Then we could replace instances of `errors.errorSummary` e.g

```
{{ govukErrorSummary({
  titleText: "There is a problem",
  errorList: validationErrors
```

Reply:
Thanks @sw-mattw I think we should keep it simple and go with the following instead - existing approach in this PR but renamed to this:
```
{% set hasValidationErrors = errors.errorSummary | length %}
```

There's some misalignment with naming e.g `errors.errorSummary`, `validationErrors.summary`, `errorSummary` so would like to keep it simple. What do you think?

Reply:
Just to add to Tony's comment -- I tend to like renaming like this [to make it clear at a glance it's a boolean](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig), although if there's a reason for this to need the word `page` i'd be interested to hear it :) `hasValidationErrorsOnPage` could be an alt name if we need it -- but I'm not sure we do

Reply:
Thing is, I don't think there'd ever be an instance where there are validation errors, _not_ on the page, in page

Reply:
Simplified to `hasErrors` as we should add the prefix when there are any errors, and at the minute all errors on these pages are likely to only be validation errors